src/midi_Settings.h

```diff
@@ -70,7 +70,7 @@ struct DefaultSettings
     termination, the receiver will turn off all voices and return to
     normal (non- active sensing) operation..
     */
-    static const bool UseSenderActiveSensing = false;
+    static const bool UseSenderActiveSensing = true;
```

Comment:
The default should probably be `false` to avoid spamming other devices. It can be an opt-in behaviour when disconnection detection is desired.

Reply:
I've been thinking about it as well, some devices have it as default (Roland, and recently a Cinara box), other don't. Happy to switch it off.